recipes/install-geolite2-db.rb

```diff
@@ -0,0 +1,36 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-ua-harvester
```

Comment:
Should that comment be #Recipe:: install-geolite2-db

Reply:
oof. yes. I also need to update `metadata.rb`. I'll fix in develop and make a new rc tag.